tests/test_sinope.py

```diff
@@ -57,3 +73,133 @@ async def test_sinope_flow_measurement(zigpy_device_from_quirk, quirk):
+    data = _get_packet_data(foundation.GeneralCommand.Report_Attributes, attr)
+    device.handle_message(260, cluster_id, endpoint_id, endpoint_id, data)
+
+    if exp_event is None:
+        assert cluster_listener.zha_send_event.call_count == 0
+    else:
+        assert cluster_listener.zha_send_event.call_count == 1
+        assert cluster_listener.zha_send_event.call_args == mock.call(
+            exp_event,
+            {
+                "attribute_id": 84,
+                "attribute_name": "action_report",
+                "value": press_type.value,
+            },
+        )
+
+
+@pytest.mark.parametrize("quirk", (SinopeTechnologieslight,))
+async def test_sinope_light_switch_non_action_report(zigpy_device_from_quirk, quirk):
+    """Test commands not handled by custom handler.
+
+    Make sure that non attribute report commands and attribute reports that don't
+    concern action_report are passed through to base class.
+    """
+
+    device: Device = zigpy_device_from_quirk(quirk)
+    cluster_id = SINOPE_MANUFACTURER_CLUSTER_ID
+    endpoint_id = 1
+
+    class Listener:
+        zha_send_event = mock.MagicMock()
+
+    cluster_listener = Listener()
+    device.endpoints[endpoint_id].in_clusters[cluster_id].add_listener(cluster_listener)
+
+    # read attributes general command
+    data = _get_packet_data(foundation.GeneralCommand.Read_Attributes)
+    device.handle_message(260, cluster_id, endpoint_id, endpoint_id, data)
+    # no ZHA events emitted because we only handle Report_Attributes
+    assert cluster_listener.zha_send_event.call_count == 0
+
+    # report attributes command, but not "action_report"
+    attr = foundation.Attribute(
+        attrid=0x10,  # "on_intensity" attribute
+        value=foundation.TypeValue(
+            type=t.int16s(0x29), value=t.int16s(50)
+        ),  # 0x29 = t.int16s
+    )
+    data = _get_packet_data(foundation.GeneralCommand.Report_Attributes, attr)
+    device.handle_message(260, cluster_id, endpoint_id, endpoint_id, data)
+    # ZHA event emitted because we pass non "action_report"
+    # reports to the base class handler.
+    assert cluster_listener.zha_send_event.call_count == 1
+
+
+@pytest.mark.parametrize("quirk", (SinopeTechnologieslight,))
+async def test_sinope_light_switch_reporting(zigpy_device_from_quirk, quirk):
+    """Test that configuring reporting for action_report works."""
+    device: Device = zigpy_device_from_quirk(quirk)
+
+    manu_cluster = device.endpoints[1].in_clusters[SINOPE_MANUFACTURER_CLUSTER_ID]
+
+    request_patch = mock.patch("zigpy.zcl.Cluster.request", mock.AsyncMock())
+    bind_patch = mock.patch("zigpy.zcl.Cluster.bind", mock.AsyncMock())
+
+    with request_patch as request_mock, bind_patch as bind_mock:
+        request_mock.return_value = (foundation.Status.SUCCESS, "done")
+
+        await manu_cluster.bind()
+        await manu_cluster.configure_reporting(
+            SinopeTechnologiesManufacturerCluster.AttributeDefs.action_report.id
```

Comment:
Add back missing comma I removed:
```suggestion
            SinopeTechnologiesManufacturerCluster.AttributeDefs.action_report.id,
```